bin/ch/Debugger.cpp

```diff
@@ -175,14 +175,14 @@ JsValueRef Debugger::Evaluate(JsValueRef callee, bool isConstructCall, JsValueRe
     {
         IfJsErrorFailLogAndRet(ChakraRTInterface::JsNumberToInt(arguments[1], &stackFrameIndex));
 
-        LPCWSTR str = nullptr;
+        AutoString argstr;
         size_t length;
-        IfJsErrorFailLogAndRet(ChakraRTInterface::JsValueToWchar(arguments[2], &str, &length));
+        IfJsErrorFailLogAndRet(ChakraRTInterface::JsValueToCharCopy(arguments[2], &argstr, &length));
 
 #ifdef _WIN32
-        ChakraRTInterface::JsDiagEvaluate(str, stackFrameIndex, &result);
+        ChakraRTInterface::JsDiagEvaluateUtf8(*argstr, stackFrameIndex, &result);
```

Comment:
Why is this under #ifdef? Can we remove the ifdef since JsDiagEvaluateUtf8 is now implemented?